src/web/mjs/connectors/templates/Tappytoon.mjs

```diff
@@ -0,0 +1,72 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class Tappytoon extends Connector {
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = [];
+        this.url = undefined;
+        this.baseURL = 'https://www.tappytoon.com';
+        this.apiurl = 'https://api-global.tappytoon.com';
+        this.lang = undefined;
+        this.requestOptions.headers.set('x-referer', this.baseURL);
+        this.requestOptions.headers.set('x-origin', this.baseURL);
+        this.requestOptions.headers.set('Accept-Language', this.lang);
+    }
+
+    get icon() {
+        return '/img/connectors/tappytoon';
+    }
+
+    async _initializeConnector() {
+        let uri = new URL(this.baseURL);
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'script#__NEXT_DATA__');
+        let obj = JSON.parse(data[0].text).props.initialState.axios.headers;
+        this.requestOptions.headers.set('Authorization', obj['Authorization']);
```

Comment:
Does these expire?
Will they still be the same when user login via manual website interaction?

Reply:
The Authorization header contains a bearer token, and yes it will change.